src/escpos/escpos.py

```diff
@@ -733,6 +738,18 @@ def panel_buttons(self, enable=True):
         else:
             self._raw(PANEL_BUTTON_OFF)
 
+    def queryStatus(self):
+        self._raw(RT_STATUS_ONLINE)
+        time.sleep(1)
+        status = self._read();
+        if (len(status) > 0):
+            return status;
+        else:
+            return [8];
+
+    def isOnline(self):
+        status = self.queryStatus()[0];
+        return ((status & RT_MASK_ONLINE) == 0);
```

Comment:
Semicolons are not needed at the end of a sentence.
Also, syntax could be more pythonic changing '&' with 'and'.

``
return ((status and RT_MASK_ONLINE) == 0)
``